app/src/main/res/layout/main_activity.xml

```diff
@@ -43,6 +43,7 @@
 
         <FrameLayout
             android:id="@+id/main"
+            android:background="@color/splashscreen_background"
```

Comment:
There's no issue with dark themes theming Substratum. We don't need to specify a color to get them to theme it, as seen on Mono, it all themes perfectly.